tensorflow/compiler/xla/service/gpu/gpu_types.h

```diff
@@ -21,10 +21,19 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
-// GpuVersion is used to abstract Gpu hardware version. On Cuda platform,
-// it comprises a pair of integers denoting major and minor version.
-// On ROCm platform, it comprises one integer for AMD GCN ISA version.
-using GpuVersion = absl::variant<std::pair<int, int>, int>;
+// GpuVersion is used to abstract Gpu hardware version.
+//
+// On Cuda platform, it comprises of an <int, int> pair
+// denoting major and minor version.
+//
+// On ROCm platform, it comprises of an <int, string> pair
+// the int has the contents of the hipDeviceProp_t::gcnArchValue field.
+// the string ahs the contents of the hipDeviceProp_t::gcnArchName field.
```

Comment:
ahs->has

Reply:
will fix this when I push out the next commit